lib/triggers/rollTrigger.js

```diff
@@ -39,12 +45,27 @@ RollTrigger.prototype._respond = function(roomId, userId, message) {
 	var that = this;
 	var params = that._stripCommand(message, that);
 	if(params) {
-		try {
-			var rolled = dice.roll(params);
-			that._sendMessageAfterDelay(toId, rolled.result.toString() + (parseInt(rolled.input.quantity) > 1 ? " ("+JSON.stringify(rolled.rolled).replace(/,/g,", ")+")" : ""));
-		} catch(err) {
-			that._sendMessageAfterDelay(toId, "Try using syntax from www.github.com/troygoode/node-roll");
-		} return true;
+		var split = params.split('d');
+		if(parseInt(split[0]) === 1) {
+			try {
+				var rolled = dice.roll(params);
+				that._sendMessageAfterDelay(toId, rolled.result.toString() + (parseInt(rolled.input.quantity) > 1 ? " ("+JSON.stringify(rolled.rolled).replace(/,/g,", ")+")" : ""));
+			} catch(err) {
+				that._sendMessageAfterDelay(toId, "Try using syntax from www.github.com/troygoode/node-roll");
+			} return true;
+		}
+		else if(parseInt(split[0]) > this.options.maxDice || parseInt(split[1]) > this.options.maxSize) {
+			this._sendMessageAfterDelay(toId, 'You cannot specify a number greater than 64 on either side of the "d".');
+			return true;
+		}
+		else {
+			try {
+				var rolled = dice.roll(params);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Variable 'rolled' is already declared in line 46.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/968187231/issues/source?bid=1980036&fileBranchId=2848073#l63)